pyvista/examples/examples.py

```diff
@@ -193,11 +193,12 @@ def load_globe():
     --------
     >>> from pyvista import examples
     >>> dataset = examples.load_globe()
-    >>> dataset.plot()
+    >>> texture = examples.load_globe_texture()
+    >>> dataset.plot(texture=texture)
 
     """
     globe = pyvista.PolyData(globefile)
-    globe.textures['2k_earth_daymap'] = load_globe_texture()
+    globe._textures['2k_earth_daymap'] = load_globe_texture()
```

Comment:
The benefit of this switch is to avoid a deprecation warning?  If so, should there be a pointer for this somewhere (maybe in DataSet.textures there can be a source code comment along the lines of `# use _textures internally to avoid PyVistaDeprecationWarning`?)  Not really sure if it is value add, but it confused me for a few minutes anyway.

Reply:
Yes, this switch is to avoid the deprecation warning while loading example data. We could, but no new code should use this API now that it is deprecated so I don't think there would be any added value to the note.